opam-ci-check/lib/lint.ml

```diff
@@ -313,7 +306,14 @@ module Checks = struct
           if is_perm_correct path then [] else [ (pkg, ForbiddenPerm path) ]
       | file -> [ (pkg, UnexpectedFile file) ]
     in
-    get_files dir |> List.map check_file |> List.concat
+    (* FIXME: Would it be better to make skipping this check more explicit? *)
+    if Sys.file_exists dir then
+      get_files dir |> List.map check_file |> List.concat
+    else (
+      print_endline
+      @@ Printf.sprintf
+           "Skipped check_package_dir since package dir %s doesn't exist" dir;
+      [])
```

Comment:
The publication tools first generate the opam metadata (OpamFile.OPAM.t) objects, and I was trying to integrate the lints at that stage, rather than after they create the required files.